ros1/gazebo/README.md

```diff
@@ -0,0 +1,45 @@
+# ROS1 Gazebo simulation demo
+
+Code reference for ROS 1 Gazebo simulation tutorial at https://foxglove.dev/blog/simulating-robotic-scenarios-with-gazebo-and-ros1.
```

Comment:
Fix link - should be `ros1-and-gazebo`.

https://github.com/foxglove/website/pull/1093/commits/cafff89153a45564cb1654a8d1bd27d0036eabc1